### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by ademibowale**
Did the testing validation for all the spec files

### CODE REVIEW

File: spec/models/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```
Comment (VICTOR-CHUKWU) [inline spec/models/post_spec.rb:1]: - [ ] Kindly include unit test for your post

File: spec/models/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```
Comment (VICTOR-CHUKWU) [inline spec/models/like_spec.rb:1]: - [ ] Kindly include unit test for like model

File: spec/models/comment_spec.rb [hunk 3]
```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```
Comment (VICTOR-CHUKWU) [inline spec/models/comment_spec.rb:1]: - [x] Kindly add unit test for comment model

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ademibowale  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/comment_spec.rb (hunk 3 above)
Comment (ademibowale) [reply spec/models/comment_spec.rb:1]: Thank you ,its done

File: spec/models/like_spec.rb (hunk 2 above)
Comment (ademibowale) [reply spec/models/like_spec.rb:1]: Thank you ,its done .

File: spec/models/post_spec.rb (hunk 1 above)
Comment (ademibowale) [reply spec/models/post_spec.rb:1]: Thank you ,its done

**Review by descholar-ceo — APPROVED** [review]
Hi @ademibowale 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requirements ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**